fixtures/braid-design-system/package.json

```diff
@@ -7,5 +7,8 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "sku": "workspace:*"
+  },
+  "devDependencies": {
+    "@vanilla-extract/css": "^1.0.0"
```

Comment:
I'm confused about this one being a devDep vs dep. `sku` is in `dependencies` but `@vanilla-extract/css` is in `devDependencies`?! Shouldn't all of them be in `dependencies`?

Reply:
Good point. Haven't been super rigorous about the distinction between dev dep and dep in fixtures. VE is usually a dev dep since it's not used at runtime, but you're right that `sku` should be a dev dep. I'll fix that up in all the fixtures.